streamz/sources.py

```diff
@@ -510,7 +517,9 @@ def checkpoint_emit(_part):
                     except (RuntimeError, ck.KafkaException):
                         continue
                     if 'auto.offset.reset' in self.consumer_params.keys():
-                        if self.consumer_params['auto.offset.reset'] == 'latest':
+                        if self.consumer_params['auto.offset.reset'] == 'latest' and \
+                                (self.positions == [-1001] * self.npartitions
```

Comment:
Where does -1001 come from?
In this expression, I would prefer parentheses to make clear which `and`/`or` goes together.

Reply:
> Where does -1001 come from?

The default value returned by `consumer.committed()` which is assigned to `self.positions` is `-1001`. 

>  parentheses to make clear which `and`/`or` goes together

There's one already there. It shows `latest` `and` (`-1001` `or` `0`). Did you mean another parenthesis around the entire expression?